src/components/ButtonPanel.js

```diff
@@ -5,36 +5,36 @@ export default function ButtonPanel() {
   return (
     <div className="button-panel">
       <div className="group group-1">
-        <Button name="AC" />
-        <Button name="+\-" />
-        <Button name="%" />
+        <Button name="AC" color="btn-gray" />
+        <Button name="+\-" color="btn-gray" />
+        <Button name="%" color="btn-gray" />
         <Button name="÷" />
       </div>
 
       <div className="group group-2">
-        <Button name="7" />
-        <Button name="8" />
-        <Button name="9" />
+        <Button name="7" color="btn-gray" />
+        <Button name="8" color="btn-gray" />
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.